libsolidity/analysis/SyntaxChecker.cpp

```diff
@@ -217,6 +219,17 @@ bool SyntaxChecker::visit(Throw const& _throwStatement)
 
 bool SyntaxChecker::visit(Literal const& _literal)
 {
+	if (_literal.token() == Token::UnicodeStringLiteral)
+	{
+		// Ensure it contains valid UTF-8 sequences only.
+		if (!validateUTF8(_literal.value()))
+			m_errorReporter.syntaxError(
+				8452_error,
+				_literal.location(),
+				"Non-Unicode characters found"
```

Comment:
Should this say "Invalid UTF-8 sequence found"?

Reply:
Yes, probably better.

Reply:
Done.